opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -83,10 +84,27 @@ def http_status_to_status_code(
 def unwrap(obj, attr: str):
```

Comment:
```suggestion
def unwrap(obj: Union[object, str], attr: str):
```
Maybe we can type this too?

Reply:
Added, thanks